src/main/lib/exclusion-utils.js

```diff
@@ -0,0 +1,35 @@
+function isDirExclusion (exclusion) {
+  return exclusion.indexOf('.') === -1;
+}
+
+function isExcluded (filePath, exclusion) {
+  let toMatch;
+  if (isDirExclusion(exclusion)) {
+    let index = filePath.lastIndexOf('/');
+    // Return if it is a top level file
+    if (index === -1) {
+      return false;
+    }
+
+    toMatch = filePath.substring(0, index);
+  } else {
+    toMatch = filePath;
+  }
+
+  const regex = new RegExp(exclusion, 'g');
+  return toMatch.match(regex) != null;
+}
+
+const prepareExclusion = (exclusion) => exclusion.replace('*', '\\w*');
```

Comment:
The is no point to make that function a public API. It's reusability is close to 0. Name is bad as it does not communicate what it does. Name properly and make it private please.